### PR TITLE
manually updated docs that should have gone with e31bc0aa6bd0d461b776…

### PR DESCRIPTION
**Pull request by dlebauer**
I made a minor edit in e31bc0aa6bd0d461b776c22f47cbd9e6e4f370bb but didn't run make document

### CODE REVIEW

**Review by mdietze — CHANGES_REQUESTED** [review]
updated file seems to be missing a closing }

**Comment by mdietze** [comment]
Current failure seems unrelated to David's changes. Any guesses @infotroph @robkooper 

```
Run docker/login-action@v3
Error: Username and password required
```

**Comment by infotroph** [comment]
All the `Docker GHA` jobs fail at the moment for PRs opened from external forks, because GitHub (correctly!) doesn't share our repository secrets with those runs (otherwise people could exfiltrate secrets just by opening PRs from forks they control). 
 
 @robkooper is working on this; not sure if there's an ETA for a fix yet. See https://pecanproject.slack.com/archives/C8RD9BRLK/p1725833043616009

bottom line: If the Docker checks are the only ones failing then it's OK to merge.

**Review by infotroph — APPROVED** [review]
